docs/person/zodiac.md

```diff
@@ -0,0 +1,12 @@
+# zodiac
+
+```js
+// usage
+chance.zodiac()
+```
+Generate a random zodiac signs.
+
+```js
+chance.zodiac();
+=> 'Bull'
```

Comment:
Can you also update the docs to match the common Zodiac signs?

Reply:
Okey, I forgot. Sorry

Reply:
Done